src/doc/gui_manual/Quantitative/LineSampler.rst

```diff
@@ -0,0 +1,331 @@
+a 2D window. For 1D a window the results will be displayed as curve
+plot in much the same way a Lineout data would be drawn.
+
+Unique and critical to using the LineSampler operator is that it
+should be defined in each window for each plot. Further the "Apply to
+all windows" and "Apply operators to all plots" should be checked. As
+such there is some critical setup that must be done.
+
+The genesis of the Line Sampler operator is from plasma physics and
+the synthethic diagnostics performed in fusion simulations so to
+compare against experimental results. As such the nomenclature is
+based on this usage.
+
+
+LineSampler operator
+~~~~~~~~~~~~~~~~~~~~
+
+To use the LineSampler mode first create a translucent Pseudocolor
+plot of the data for reference. Clone the plot and add the LineSampler
+Operator to the plot. Remember to set the Apply to "active window" and
+uncheck "Apply operators to all plots" before adding the LineSampler
+operator.  Next bring up the LineSampler operator attribute window.
+
+Main tab
+""""""""
+
+Select the "Main" tab, :numref:`Figure %s <LineSampleMainTab>` set up
+the attributes accordingly.
+
+
+1. The line sampler needs to know the mesh geometry which can be
+   Cartesian, Cylindrical, Toroidal which determines how the sampling
+   will be done. Most notable is that the differetnce between
+   cylindrical and toroidal coordinates is coordiante ordering r, phi,
+   z vs r, z, phi.
+
+2. The user can set up an array configuration specifying the geometry
+   or manually read in a channel configuration file. The "Geometry"
+   and "List" tabs reflect those respective choices.
+
+3. The boundary of the sampling can be based on the data file or using
+   a manually specified wall file. The wall file reflects the boundary
+   in a 2D slice of the data in the Y=0 or the Phi=0 plane. Any
+   sampling outside of the boundary will be disregarded, that is the
+   sample ray will be clipped.
+
+   Note currently the wall file format is specific to D3D fusion
+   files. The coordinates are displayed in the text box.
+
+4. When using the LineSampler operator, one will need to check the
+   "Apply operators to all plots" option. So that all of the
+   parameters are propagated to al of the windows. However, the user
+   may want to have a different Instance" of the operator so to
+   compare different sampling configurations. So by setting the
+   "Instance" to 'A' for the first, and 'B' for the second the
+   attributes will be propagated appropriately to the plots in other
+   windows when using the "Apply to all windows" option.
+
+.. _LineSampler_MainTab:
+
+.. figure:: images/LineSamplerMainTab.png
+
+   LineSampler Main Tab
+
+Geometry Tab
+""""""""""""
+
+The Geometry tab is active only if the "Array configuration" is set to
+"Geometry." This tab allows the user to manualy define one or more
+arrays with one or more channels.
+
+1. The number of arrays to be created.  Each array consists of mutlple
```

Comment:
Remove one of the spaces at the beginning of the second sentence. Change "mutlple" to "multiple".